docs/tutorials/measurement.md

```diff
@@ -1,105 +1,113 @@
-# 測定
+# Measurement with CARET
 
-サンプルのアプリケーションを例に、ツールの使い方を説明していきます。
+This page explains usage of CARET with a sample application.
+The sample application is located on [CARET_demos](https://github.com/tier4/CARET_demos.git) repository.
 
-サンプルのリポジトリ<https://github.com/tier4/CARET_demos.git>
+## Building application with CARET
 
-## アプリケーションのビルド
+To measure target applications' performance, the target should be built with CARET/rclcpp. CARET/rclcpp is a fork of [ROS 2-based rclcpp](https://github.com/ros2/rclcpp) which has some additional tracepoints defined by CARET. If you have already built the target without CARET/rclcpp, you have to build the target with CARET/rclcpp again.
 
-caret の環境変数を`/opt/ros/galactic/setup.bash`の後に設定し、測定対象のアプリケーションをビルドします。  
-caret 導入前にビルド済みのアプリケーションも、再度ビルドが必要になります。
+For building the application with CARET/rclcpp, CARET's `local_setup.bash` should be applied along with ROS 2's `setup.bash` as shown below.
 
 ```bash
 mkdir -p ~/ros2_ws/src
 cd ~/ros2_ws
 
 git clone https://github.com/tier4/CARET_demos.git src/CARET_demos --recursive
-source ~/ros2_caret_ws/install/local_setup.bash
+source /opt/ros/galactic/setup.bash
+source ~/ros2_caret_ws/install/local_setup.bash # please keep the order after 'source /opt/ros/galactic/setup.bash'
 
 colcon build --symlink-install --packages-up-to caret_demos
 ```
 
-> 測定対象のアプリケーションも再ビルドが必要な理由
->
-> caret は rclcpp のヘッダーで読み込まれるテンプレート実装内にもトレースポイントを追加しています。  
-> トレースポイントを追加された rclcpp を有効にするため、測定対象のアプリケーションを再度ビルドする必要が有ります。  
-> 従って、 apt でインストールした demo_nodes_cpp などのアプリケーションは測定することが出来ません。  
-> 測定するにはソースファイルからビルドし直す必要があります。
+> Reason to build the target with CARET/rclcpp
+> Some tracepoints must be added to template implementation, which is referred by rclcpp, for CARET to measure performance.
```

Comment:
which is referred by rclcpp header files

Reply:
Fixed in https://github.com/tier4/CARET_doc/pull/36/commits/7331b7dbcec4edbb06e9a15fcba6806856201826